api/guides/filtering.md

```diff
@@ -4,31 +4,79 @@ title: Filtering
 
 # Filtering
 
-At times it's desirable to request a subset of records by means of a filter. Some API endpoints support the ability to filter by specific attributes. Filterable attributes will be noted on their respective index endpoint documentation areas.
+When using listing (GET) endpoints in the Adobe Experience Platform Launch API, you may find it necessary to limit the returned results to a subset of records. To accomplish this, many of the API's listing endpoints support the ability to filter by specific attributes.
 
-To filter records on an index endpoint, supply a `filter` parameter in a request
-URL, formatted as: `filter[FILTERABLE_FIELD]=value`. Filter values must be exact
-matches.
+{% alert info, Note %}
+Filters are primarily used to limit result for listing (GET) endpoints. If you wish to make structured queries to the API instead, see the guide on [searching](/api/guides/searching/).
```

Comment:
I think the styling of the alert will make this stand out a lot more than we need it to.  Maybe just add the second sentence "If you wish...." to the paragraph above and remove the alert?